docs/roadmap.md

```diff
@@ -32,6 +33,14 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 #### Observability
 
+- [ ] Logging console enhancement
+- [ ] Monitoring stack upgrade including Prometheus, Prometheus Operator, Node exporter, kube-state-metrics etc.
+- [ ] Custom metrics support including application custom metrics dashboard, custom metrics HPA
+- [ ] Integration with Alertmanager
+- [ ] K8s Event management
+- [ ] K8s Audit Support
```

Comment:
Any more information or reference regarding K8s Event and Audit? Do you plan to integrate native upstream k8s event and audit feature, will any new KS APIs to add? @benjaminhuo